src/thread/model.rs

```diff
@@ -390,7 +391,7 @@ impl Threads {
             .save(&mut transaction)
             .await?;
 
-        if payload.subscribe {
+        if payload.subscribe && uuid_data.discriminator != "user" {
```

Comment:
Does this make sense? Shouldn't I still get notifications for threads created at user profiles (or wherever those are displayed)?